src/index.rs

```diff
@@ -1199,6 +1199,10 @@ impl Index {
             .collect::<Vec<_>>()
     }
 
+    pub fn has_variadic_parameter(&self, pout_name: &str) -> bool {
+        self.get_pou_members(pout_name).iter().any(|member| member.is_parameter() && member.is_variadic())
```

Comment:
I am guessing this is a typo?
```suggestion
    pub fn has_variadic_parameter(&self, pou_name: &str) -> bool {
        self.get_pou_members(pou_name).iter().any(|member| member.is_parameter() && member.is_variadic())
```